src/themes/teams/components/Text/textStyles.ts

```diff
@@ -21,7 +21,7 @@ export default {
       temporary,
     },
     variables: v,
-  }: ComponentStyleFunctionParam<TextProps, TextVariables>): ICSSInJSStyle => {
+  }: ComponentStyleFunctionParam<TextProps, TeamsTextVariables>): ICSSInJSStyle => {
```

Comment:
Remove parts which are already handled by base theme. Teams theme is merged on top of base theme, it should only (re)define variables and overwrite styles where necessary.

Reply:
I removed the duplicated code from styles side. As for variables, the color palettes and the font sizes are different, so keeping the changes in the teams theme as well.